<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>preferences</class>
 <widget class="QMainWindow" name="preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>606</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PREFERENCES</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qlineargradient(
    x2:0, 
	y1:0, 
	x1:0, 
	y1:1,
    stop:0 #404040,
    stop:1 #E6E6E6
);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>0</y>
      <width>841</width>
      <height>581</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>180</y>
       <width>281</width>
       <height>311</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btn_mentor_meeting">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #D96B6B;
    color: white;
    border-radius: 8px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #E07C7C;
}

QPushButton:pressed {
    background-color: #B85A5A;
}

QPushButton:checked {
    background-color: #A94444;
    font-weight: bold;
}</string>
        </property>
        <property name="text">
         <string>MENTOR MEETINGS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_applications">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #D96B6B;
    color: white;
    border-radius: 8px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #E07C7C;
}

QPushButton:pressed {
    background-color: #B85A5A;
}

QPushButton:checked {
    background-color: #A94444;
    font-weight: bold;
}</string>
        </property>
        <property name="text">
         <string>APPLICATIONS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_interviews">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #D96B6B;
    color: white;
    border-radius: 8px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #E07C7C;
}

QPushButton:pressed {
    background-color: #B85A5A;
}

QPushButton:checked {
    background-color: #A94444;
    font-weight: bold;
}</string>
        </property>
        <property name="text">
         <string>INTERVIEWS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_main_menu">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #D96B6B;
    color: white;
    border-radius: 8px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #E07C7C;
}

QPushButton:pressed {
    background-color: #B85A5A;
}

QPushButton:checked {
    background-color: #A94444;
    font-weight: bold;
}</string>
        </property>
        <property name="text">
         <string>MAIN MENU</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_exit">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #D96B6B;
    color: white;
    border-radius: 8px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #E07C7C;
}

QPushButton:pressed {
    background-color: #B85A5A;
}

QPushButton:checked {
    background-color: #A94444;
    font-weight: bold;
}</string>
        </property>
        <property name="text">
         <string>EXIT</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>391</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
